scripts/taco_to_burrito.py

```diff
@@ -32,25 +33,30 @@ def main():
             convert_markers(full_path, target_path)
 
 ################################################################################
-# Quick error printing logic that only prints the title of a section if there
-# are any errors in it. Could be handled better by returning errors from the
-# function but that is more work to implement then this is.
+# Quick error printing logic
 ################################################################################
-eprint_title: str = ""
-eprint_trigger: bool = False
-def set_eprint_title(title: str):
-    global eprint_title
-    global eprint_trigger
-    eprint_title = title
-    eprint_trigger = False
-def eprint(*args, **kwargs):
-    global eprint_trigger
-    if not eprint_trigger:
-        eprint_trigger = True
-        print(eprint_title)
-    print("   ", *args, **kwargs)
-
-
+error_location = ""
+class ErrorType(Enum):
+    INFO = 0
+    WARNING = 1
+    ERROR = 2
+    
+def eprint(*args):
```

Comment:
You can probably change this to
```python
def eprint(error_level, *args, **kwargs)
```
To get rid of the `for arg in list(args):` loops. If you want to assign a default value to it there might be some additional changes I would have to experiment with to find out, eg: it might require you to do `def eprint(*args, error_level=ErrorType.WARNING, **kwargs)` instead.

Reply:
i was getting some weird behavior earlier when i was trying this approach instead. seems like moving the error_level argument to after *args solves it, and lets me set a default value.